files/sanity_check.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
```

Comment:
Yes it does :) Not [100% coverage](https://github.com/open-craft/ansible-sanity-checker/blob/jbzdak/sanity-checker/tests/test_sanity_check.py) but covers all checks --- it would be rather bad if they failed :) If there is any configuration issue, this most probably would fail deployment, as sanity check is run at every deployment to check config.